flutter/example/lib/main.dart

```diff
@@ -14,16 +15,30 @@ Future<void> main() async {
   await SentryFlutter.init(
     (options) {
       options.dsn = _exampleDsn;
-      // use breadcrumb tracking of WidgetsBindingObserver
-      // options.useFlutterBreadcrumbTracking();
-      // use breadcrumb tracking of platform Sentry SDKs
-      // options.useNativeBreadcrumbTracking();
+      if (UniversalPlatform.isAndroid) {
+        options.addEventProcessor(addAndroidDeviceId);
+      }
     },
     // Init your App.
     appRunner: () => runApp(MyApp()),
   );
 }
 
+FutureOr<SentryEvent> addAndroidDeviceId(
+  SentryEvent event, {
+  dynamic hint,
+}) async {
+  var info = await DeviceInfoPlugin().androidInfo;
+  var user = (event.user ?? SentryUser()).copyWith(id: info.fingerprint);
+
+  // todo there's no ID field on SentryDevice
+  // var device = (event.contexts.device ?? SentryDevice()).copyWith();
```

Comment:
that's ok, it's not gonna affect the number of unique users or anything, so its not a breaking change as it does not even exist in the Dart/Flutter SDK